sqlagg/base.py

```diff
@@ -175,6 +179,12 @@ def table_filter(table_name, metadata):
 
         return self._metadata
 
+    def count(self, connection, filter_values=None):
```

Comment:
those docs are for an ORM query which this is not but guessing it just works. A unit test would be useful though.